scoring/static/scss/scoring.scss

```diff
@@ -57,4 +57,22 @@
 @import "../bootstrap/utilities";
 @import "../bootstrap/print";
 
-@import "mysociety-standard-footer";
+//The variables.scss is not being imported so I'm putting some variables here.
```

Comment:
@struan can you help @lucascumsille work out what’s going wrong here with the Sass includes? It‘s either something to do with a conflict between caps/scoring filenames and bootstrap filenames, or a conflict between caps and scoring filenames… or we’re just not using the correct relative path when importing.

Reply:
it's importing fine, some of the colours defined in scoring.scss aren't in variables which seem to be the root of the problem. if you add

```
$color-league-green-l1: #E4FAF5;
$color-league-green-d1: #0BC7A0;
$color-league-blue-l2: #D8F2FE;
$color-league-blue-l1: #7BD2F8;
$color-league-blue-d1: #045C88;
```

to variables then everything seems to work fine.